e2e/test/scenarios/collections/collection-items-listing.cy.spec.js

```diff
@@ -238,6 +235,9 @@ describe("scenarios > collection items listing", () => {
 function toggleSortingFor(columnName) {
   const testId = "items-table-head";
   cy.findByTestId(testId).findByText(columnName).click();
+  // These random waits are really discouraged, but I couldn't find a reliable
+  // way to prevent the flaky sorting test without it.
+  cy.wait(50);
```

Comment:
This is an actual fix.
The rest is the whitespace diff after cd56ce7903de60dbcc0ed4c365a5a7261927ad85